pkg/simple/client/k8s/kubernetes.go

```diff
@@ -35,13 +36,19 @@ func NewKubernetesClientOrDie(options *KubernetesOptions) *KubernetesClient {
 
 	config.QPS = options.QPS
 	config.Burst = options.Burst
-
+	var master string
```

Comment:
Agreed. But I think it's better to move this section to the end of this function, since we assign options.Master to instance `k`, also need to check options.Master missing scheme or not